OrbitGl/ProcessesDataView.cpp

```diff
@@ -81,8 +81,9 @@ void ProcessesDataView::DoSort() {
 
 #undef ORBIT_PROC_SORT
 
-void ProcessesDataView::OnSelect(int index) {
-  const ProcessInfo& selected_process = GetProcess(index);
+void ProcessesDataView::OnSelect(std::optional<int> index) {
+  CHECK(index.has_value());
```

Comment:
Same comment as for the LiveFunctionsView, I think this must be handled more gracefully. This is a case that could happen in some circumstances, and the CHECK will crash Orbit.